js-code-guidelines.md

```diff
@@ -395,7 +397,7 @@ Every module below 1.0.0 should use `~` instead of `^`.
 
 ##### Pre-Commit
 
-[precommit](https://www.npmjs.com/package/pre-commit) helps us check code style run the tests on every commit. In your `package.json`:
+[pre-commit](https://www.npmjs.com/package/pre-commit) helps us check code style run the tests on every commit. In your `package.json`:
```

Comment:
Thank for fixing. Though this should really be "pre-push" instead. But that is I guess a separate commit.